projects/e2e/workbench/src/router-link.e2e-spec.ts

```diff
@@ -0,0 +1,46 @@
+/*
+ * Copyright (c) 2018-2019 Swiss Federal Railways
+ *
+ * This program and the accompanying materials are made
+ * available under the terms of the Eclipse Public License 2.0
+ * which is available at https://www.eclipse.org/legal/epl-2.0/
+ *
+ *  SPDX-License-Identifier: EPL-2.0
+ */
+
+import { browser, Key } from 'protractor';
+import { WelcomePagePO } from './page-object/welcome-page.po';
+import { TestcaseBb9700a6ViewPO } from './page-object/testcase-bb9700a6-view.po';
+import { AppPO } from './page-object/app.po';
+
+describe('RouterLink', () => {
+
+  const appPO = new AppPO();
+  const welcomePagePO = new WelcomePagePO();
+  const viewPO = new TestcaseBb9700a6ViewPO();
+
+  beforeEach(async () => {
+    await browser.get('/');
+    await welcomePagePO.clickTile('e2e-tile-bb9700a6');
+  });
+
+  it('should open view navigation in current view [testcase: bb9700a6-view]', async () => {
```

Comment:
See the comment above of navigating to `/view` instead of `/view-navigation`. Please change the should-texts accordingly, e.g., `should open the testing view in the current view  [testcase: bb9700a6-view]`.

Reply:
done